linkml_model/model/schema/meta.yaml

```diff
@@ -109,9 +109,37 @@ slots:
     in_subset:
       - owl
 
+  alias_source:
+    range: uriorcurie
+    required: true
+    description: >-
+      Can be a URIOrCurie that represents the source of the alias, or it can be a simple string that
+      describes the source of the alias.
+
+  alias_scope:
+    range: alias_scope_enum
+    description: >-
+      The relatedness of an alias to the element it is aliasing.  alias_scope can be: narrow, broad, exact,
+      related, close, etc.
+
+  alias_name:
+    range: string
+    required: true
+    exact_mappings: skos:altLabel
+    description: >-
+      inherits name from element class, but in the alias_definition class, this name represents
+      a skos:altLabel
+
+  structured_aliases:
```

Comment:
We need to also add this as a class_slot - I think under `element` is the right place, as that already has `aliases`

Reply:
done